### PR TITLE
make "create cluster from template" look like a mini wizard

### LINKED ISSUE #4009: Make Create from Cluster Template look like a mini wizard
**What would you like to be added? / User Story**: Make Create from Cluster Template look like a mini wizard (like add external cluster) 1) Select Template ---- 2) Create Clusters

Taken from https://github.com/kubermatic/dashboard/issues/3936 as this is bigger change.

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
make "create cluster from template" look like a mini wizard

### Which issue(s) this PR fixes

Fixes #4009

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
PTAL @maciaszczykm @floreks

**Comment by kgroschoff** [comment]
PTAL @maciaszczykm @floreks

File: src/app/shared/components/add-cluster-from-template-dialog/steps/template/template.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+-->
+
+<form [formGroup]="form"
+      fxLayout="column"
+      id="select-template-form">
+  <km-autocomplete label="Cluster Template"
+                   [formControlName]="control.ClusterTemplate"
+                   [isLoading]="isLoadingTemplates"
+                   [options]="templateNames"
+                   required="true">
```
Comment (maciaszczykm) [inline src/app/shared/components/add-cluster-from-template-dialog/steps/template/template.html:24]: s/required="true"/required

**Comment by maciaszczykm** [comment]
Zrzut ekranu 2022-02-7 o 14 42 59

The scrollbar is misplaced compated to the current version from dev:

Zrzut ekranu 2022-02-7 o 14 43 38

File: src/app/shared/components/cluster-from-template/content/component.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  Replicas = 'replicas',
+}
+
+@Component({
+  selector: 'km-cluster-from-template-content',
+  templateUrl: './template.html',
+  styleUrls: ['./style.scss'],
+  animations: [shrinkGrow],
+})
+export class ClusterFromTemplateContentComponent implements OnInit, OnDestroy {
```
Comment (maciaszczykm) [inline src/app/shared/components/cluster-from-template/content/component.ts:35]: Can we change the naming to remove the "content" part?

File: src/app/shared/components/add-cluster-from-template-dialog/style.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// limitations under the License.
+
+@use 'mixins';
+
+mat-dialog-content .km-dialog-warning {
+  margin-top: 0;
+}
+
+mat-dialog-actions {
+  min-height: unset;
```
Comment (maciaszczykm) [inline src/app/shared/components/add-cluster-from-template-dialog/style.scss:22]: Is it required?

**Comment by kgroschoff** [comment]
@maciaszczykm Good catches. Changed. 

scroll

**Pull request merged**